plenum/test/node_catchup/test_node_catchup_with_connection_problem.py

```diff
@@ -174,7 +174,24 @@ def unpatch_after_call(proof):
                                   config=tconf,
                                   ha=nodeHa, cliha=nodeCHa,
                                   pluginPaths=allPluginsPath)
-    # patch canProcessConsistencyProof
+
+    original_process_cp = node_to_disconnect.ledgerManager.canProcessConsistencyProof
```

Comment:
This is still `canProcessConsistencyProof` but need to be `processConsistencyProof`, right?